### PR TITLE
Update GitHub Actions

### PR DESCRIPTION
**Pull request by cclauss**
`CI` is ___always___ defined in GitHub Actions.
* https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

### CODE REVIEW

**Comment by ronyeh** [comment]
Hello! Thanks @cclauss 

I'm not an expert on GitHub Actions. Can you explain what each line that you patched means? (Since it's only 3 lines.) This will help future people who come upon this PR.

Thanks, and I will be happy to merge once you respond to this thread.

File: .github/workflows/ci-workflow.yaml [hunk 1]
```diff
@@ -8,15 +8,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 18.x]
```
Comment (cclauss) [inline .github/workflows/ci-workflow.yaml:11]: Let's run two tests in parallel to keep up with https://github.com/nodejs/Release#nodejs-release-working-group

File: .github/workflows/ci-workflow.yaml [hunk 2]
```diff
@@ -8,15 +8,13 @@ jobs:
...
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```
Comment (cclauss) [inline .github/workflows/ci-workflow.yaml:14]: To keep up with https://github.com/actions/checkout/releases

File: .github/workflows/ci-workflow.yaml [hunk 3]
```diff
@@ -8,15 +8,13 @@ jobs:
...
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
```
Comment (cclauss) [inline .github/workflows/ci-workflow.yaml:16]: To keep up with https://github.com/actions/setup-node/releases

File: .github/workflows/ci-workflow.yaml [hunk 4]
```diff
@@ -8,15 +8,13 @@ jobs:
...
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm test
-        env:
-          CI: true
```
Comment (cclauss) [inline .github/workflows/ci-workflow.yaml:22]: This is not needed as discussed in the commit message.

File: .github/workflows/ci-workflow.yaml [hunk 5]
```diff
@@ -7,16 +7,15 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
```
Comment (cclauss) [inline .github/workflows/ci-workflow.yaml:10]: Run both tests to the end to see which pass and which fail.

**Comment by rvilarl** [comment]
I approved to run the tests and build 18 fails.

File: package.json [hunk 6]
```diff
@@ -88,7 +88,7 @@
     "@types/node": "^18.11.11",
     "@typescript-eslint/eslint-plugin": "^5.8.1",
     "@typescript-eslint/parser": "^5.8.1",
-    "canvas": "^2.8.0",
+    "canvas": "^2.11.0",
```
Comment (cclauss) [inline package.json:91]: https://github.com/Automattic/node-canvas/releases

**Review by rvilarl — APPROVED** [review]

**Review by ronyeh — APPROVED** [review]

**Comment by ronyeh** [comment]
Thanks @cclauss

**Pull request merged**